Homework4/Task2.py

```diff
@@ -0,0 +1,20 @@
+def get_ranges(numbers):
+    """Curtails non-empty list of non-repeating integers
+
+    sorted in ascending order
+    """
+    curtail_numbers = ''
+    index = 0
+    while index < len(numbers) - 1:
```

Comment:
good solution! 
the only thing it looks a bit heavyweight and maybe we can refactor it a bit 
(i.e. rename vars, prepare strings to concat in another line... (it's not usually used a += expression but a += value))